guide/src/design/import-customization.md

```diff
@@ -168,3 +168,25 @@ possibilities!
 
   All of these functions will call `console.log` in JS, but each identifier
   will have only one signature in Rust.
+
+* `extends = Bar` - this can be used to say that an imported type extends
```

Comment:
This is going to need to move to `guide/src/reference/attributes/on-js-imports/extends.md` and a corresponding entry in the SUMMARY.md table of contents